Book_Summarizer/extractive_summarizer.py

```diff
@@ -0,0 +1,225 @@
+# extractive summarizer
+# create an extractive summary from chapters of a large document
+
+import shutil
+import os
+from sumy.parsers.plaintext import PlaintextParser
+from sumy.nlp.tokenizers import Tokenizer
+from sumy.summarizers.lex_rank import LexRankSummarizer
+from sumy.summarizers.luhn import LuhnSummarizer
+from sumy.summarizers.lsa import LsaSummarizer
+from sumy.summarizers.text_rank import TextRankSummarizer
+from sumy.models import TfDocumentModel
+from sumy.evaluation import cosine_similarity
+from sumy.evaluation import rouge_n, rouge_l_sentence_level, rouge_l_summary_level
+import pathlib
+import pandas as pd
+import csv
+
+# divide_book_into_chapters(book_id)
+#
+# divides the book file into separate chapter files
+# book file is in the data/books folder
+# chapter files are saved in the data/book_chapters folder
+# assumes chapter breaks occur when there are two empty lines in a row
+# makes chapters at least 20 lines long
+# as there are often two double spaces at the start of a chapter
+# limits chapters to the end of the next paragraph after 3000 lines
+def divide_book_into_chapters(book_id):
+    book_filename = '../data/books/' + str(book_id) + '.txt'
+    chapter_filename = '../data/book_chapters/' + str(book_id) + '-'
+    count_chapters = 0
+    count_lines_in_chapter = 0
+    previous_blank_line = False
+    book = open(book_filename, 'r', encoding='latin-1')
+    book_chapter = open(chapter_filename + str(count_chapters) + '.txt', 'w')
+    for l in book:
+        if (count_lines_in_chapter < 3000) and ((len(l) > 1) or (count_lines_in_chapter < 20)):
+            previous_blank_line = False
+            count_lines_in_chapter += 1
+            book_chapter.write(l)
+        elif (len(l) == 1) and ((previous_blank_line == True) or (count_lines_in_chapter > 3000)):
```

Comment:
minor: you probably want `>= 3000`

Reply:
Thanks for catching that, I'll change it in the next version.